### DISCUSSION TITLE
How to add recipes of Cooking Pot?

### DISCUSSION THREAD

**Discussion by qianz1** [vectorwing/FarmersDelight#464] [Q&A] [answered]
1.16.5
How to add recipes of Cooking Pot?
thanks!

**Comment by vectorwing** [comment, marked as answer]
Hi there! 👋 

Cooking recipes (and most other recipe types) are added through **datapacks**, Minecraft's way of customizing some game aspects. They can be finicky to set up, but there are a few tutorials out there which explain the process. Videos are good, but the MC wiki has a guide too:
https://minecraft.fandom.com/wiki/Tutorials/Creating_a_data_pack

Here's a wiki page detailing how to write a cooking recipe:
https://github.com/vectorwing/FarmersDelight/wiki/Cooking-Pot-Recipes

Hope it helps! ^^

**Reply by qianz1** [reply]
Okay,I get it.Thank you.
